### ISSUE TITLE
Build setup

### ISSUE THREAD

**Issue by Alizter** [ejgallego/coq-lsp#51]
1. Remove vendored submodules
2. Use coq-universe as a playground for examples

**Comment by ejgallego** [comment]
Actually coq-universe would be the main developer workspace then.

**Comment by ejgallego** [comment]
So far this is a big problem and the only solution I could think of is as follows:

For now, the tree setup (in jscoq, here) looks like:
```
Makefile
dune-project
....
fleche/
vendor/coq
vendor/coq-serapi
```

but ideally the setup would look like
```
workspace-config.yml
Makefile
coq-lsp
coq
coq-serapi
```

where coq-lsp knows nothing about what's vendored, but some tool will read workspace-config.yml , checkout the projects at the right version, and setup a Makefile.

**Comment by Alizter** [comment]
Actually, having a vendor/ directory where a user can plop symlinks to their own projects in would be quite a good workflow.

**Comment by ejgallego** [comment]
Yes, in jsCoq we do something similar, so what we are leaning towards is a system where the submodules are optional.
